src/lib/blueprint.ts

```diff
@@ -51,6 +52,13 @@ export interface Resource {
   draftMessage: string
 }
 
+interface EventResource extends Resource {
+  resourceType: 'event'
+  eventType: string
+  routePath: string
+  targetResourceType: string
```

Comment:
I realized not all events have / will have a target resource.

```suggestion
  targetResourceType: string | null
```

To implement this, we just need to check if any of the resources are in the type, e.g., `eventType.split('.').includes(resourceName)`

Reply:
What about cases like `thermostat.*` events, shouldn't `targetResourceType` be `device` for them?

Reply:
Maybe, but I'm honestly not sure yet (for reasons we can discuss elsewhere). For now targetResourceType can be null unless the event explicitly has the resource in the type.